Example/Backpack/ViewControllers/CalendarViewController.swift

```diff
@@ -37,13 +37,33 @@ class CalendarViewController: UIViewController, BPKCalendarDelegate {
     var highPrice = BPKCalendarPriceLabelCellData(price: "£480", labelStyle: BPKCalendarPriceLabelStyle.negative)
     var noPrice = BPKCalendarPriceLabelCellData(price: "–", labelStyle: BPKCalendarPriceLabelStyle.noData)
 
+    var singleSelectionConfiguration = BPKCalendarSelectionConfigurationSingle(
+        selectionHint: "Double tap to select date"
+    )
+    var multipleSelectionConfiguration = BPKCalendarSelectionConfigurationMultiple(
+        selectionHint: "Double tap to select date",
+        deselectionHint: "Double tap to deselect date"
+    )
+    var rangeSelectionConfiguration = BPKCalendarSelectionConfigurationRange(
+        firstSelectionHint: "Double tap to select departure date",
+        secondSelectionHint: "Double tap to select return date",
+        firstSelectionState: "Selected as departure date",
+        secondSelectionState: "Selected as return date",
+        betweenSelectionState: "Between departure and return date",
+        firstAndSecondSelectionState: "Selected as both departure and return date",
+        returnDatePrompt: "Now please select a return date"
+    )
+
     override func viewDidLoad() {
+        // I guess it shouldn't be possible to create a calendar without a selection config 🤔
```

Comment:
Should we add a constructor argument to `BPKCalendar` for an instance of `BPKCalendarSelectionConguration`?

Reply:
Yes that seems reasonable, we can no longer create an instance that acts as a default. It will complicated the RN code(it will have to hardcode an english string I think).